lessons/05-html-linking/about.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title>
+            ABOUT
+        </title>
+    </head>
+    <body>
+        <h1>ABOUT</h1>
+        <ul>
+            <li><a href="Home.html">Home</a></li>
+            <li><a href="News.html">News</a></li>
+            <li><a href="About.html">About</a></li>
+            <li><a href="Contact.html">Contact</a></li>
```

Comment:
Be careful with capitalization 😄 your files are named `about.html` but are referenced here as `About.html`